src/postpredict/dependence.py

```diff
@@ -82,40 +75,47 @@ def transform(self, model_out: pl.DataFrame,
         they reflect the estimated temporal dependence structure.
         """
         # pivot model_out from long to wide format
-        wide_model_out = self._pivot_horizon(model_out, reference_time_col,
-                                             horizon_col, idx_col, pred_col)
-        min_horizon = model_out[horizon_col].min()
-        max_horizon = model_out[horizon_col].max()
+        wide_model_out = self._pivot_horizon(model_out)
```

Comment:
`reference_time_col`, `horizon_col`, `idx_col`, and `pred_col` are now saved as attributes of the object, so we no longer need to pass them around as arguments and we use `self.horizon_col`, etc.